### ISSUE TITLE
AV1500: Invalid statement count for primary constructor with captured parameters

### ISSUE THREAD

**Issue by bkoelman** [bkoelman/CSharpGuidelinesAnalyzer#155]
#### Analyzer

AvoidMemberWithManyStatementsAnalyzer

#### Repro steps
```c#
class C(string s) : B(s)
{
    void M1()
    {
        ; ; ; ;
    }

    void M2()
    {
        ; ; ; ;
    }
}

abstract class B
{
    protected B(string s)
    {
    }
}
```

#### Expected behavior
No warning, because both methods contain less than 8 statements.

#### Actual behavior
A warning is reported.
